plugins/thumbnails/server/worker.py

```diff
@@ -157,3 +159,20 @@ def attachThumbnail(file, thumbnail, attachToType, attachToId, width, height):
     }
 
     return ModelImporter.model('file').save(thumbnail)
+
+
+def getImage(extension, data):
+    """
+    Check extension of image and opens it.
+
+    :param extension: The extension of the image that needs to be opened.
+    :param data: The image file stream.
+    """
+    if 'dcm' in extension:
+        # Open the dicom image
+        dicomData = dicom.read_file(six.BytesIO(data))
+        imageData = dicomData.pixel_array
+        return Image.fromarray(imageData).convert("I")
```

Comment:
Hm. As far as I understand, dicom.pixel_array should be consistent. [link](http://pydicom.readthedocs.io/en/stable/working_with_pixel_data.html)

Also, the convert doesn't assume a specific pixel format that its converting from, so it should work regardless.. right?